main.tf

```diff
@@ -47,6 +47,13 @@ resource "azurerm_role_assignment" "grant_reader_role_to_managementgroup" {
 resource "time_sleep" "wait_time" {
   create_duration = var.wait_time
   depends_on      = [azurerm_role_assignment.grant_reader_role_to_subscriptions]
+  triggers = {
+    # Save the time we first heard about the application ID, so we wait again if it changes
+    app_id = local.application_id
+    # Same thing for LW integration name, which changed for v1.0, and waiting is needed to avoid API error when recreating it too fast
```

Comment:
nitpick: Could you make this comment 'standalone' and not refer to another comment above.